src/app/project/documents/documents-tab.component.ts

```diff
@@ -166,10 +170,23 @@ export class DocumentsTabComponent implements OnInit, OnDestroy {
               });
             }
 
+            if (this.projectPhases.length === 0) {
+              res.documentsTableRow[0].searchResults.map(item => {
+                switch (item.type) {
```

Comment:
can we get rid of these switches please?

Reply:
Switches removed, and merged the statements together into a single map.